includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -198,21 +200,30 @@ public static function get_invalid_url_validation_errors( $post, $args = array()
 				continue;
 			}
 			$term = get_term_by( 'slug', $stored_validation_error['term_slug'], AMP_Validation_Error_Taxonomy::TAXONOMY_SLUG );
-			if ( ! $term ) {
-				continue;
+
+			$forced_sanitization = null;
+			if ( $args['check_forced_sanitization'] ) {
+				$error = $stored_validation_error['data'];
+				unset( $error['sources'] );
+
+				/** This filter is documented in amp/includes/validation/class-amp-validation-manager.php */
+				$forced_sanitization = apply_filters( 'amp_validation_error_sanitized', $forced_sanitization, $error );
 			}
-			if ( $args['ignore_accepted'] && AMP_Validation_Error_Taxonomy::VALIDATION_ERROR_ACCEPTED_STATUS === $term->term_group ) {
+
+			if ( $args['ignore_accepted'] && ( true === $forced_sanitization || ( $term && AMP_Validation_Error_Taxonomy::VALIDATION_ERROR_ACCEPTED_STATUS === $term->term_group ) ) ) {
```

Comment:
Actually, the `$term->term_group` property is always set because it is defined on the `WP_Term` class:

https://github.com/WordPress/wordpress-develop/blob/a26c24226c6b131a0ed22c722a836c100d3ba254/src/wp-includes/class-wp-term.php#L43-L49